src/webview/MessageListWeb.js

```diff
@@ -15,15 +15,32 @@ import { base64Utf8Encode } from '../utils/encoding';
 export default class MessageListWeb extends Component<Props> {
   context: Context;
   props: Props;
+  isWebViewReady: boolean;
   webview: ?Object;
-  isReady: boolean;
   unsentMessages: WebviewInputMessage[] = [];
 
   static contextTypes = {
     styles: () => null,
     theme: () => null,
   };
 
+  /*
+  *
+  * Synchronizing setup with webview
+  *
+  */
+  sendTestPing = (): void => {
+    if (this.isWebViewReady) {
+      return;
+    }
+    this.sendMessages([{ type: 'hello' }]);
+    setTimeout(this.sendTestPing, 10);
```

Comment:
Another point about `setTimeout`, we should use `setInterval` instead.
https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setInterval

Reply:
To illustrate this, I did change our `waitForBridge` in #3088